app/assets/stylesheets/pregnancies.scss

```diff
@@ -16,8 +16,12 @@ $font-size: 17px;
   }
 }
 
-#menu {
-  border-right: 1px solid black;
+#sections {
+  border-left: 1px solid black;
+  padding-left: 15%;
```

Comment:
I was trying to maintain the white space that you had here, here's what it looks like with no padding.
![screen shot 2017-07-03 at 10 35 55 am](https://user-images.githubusercontent.com/4335814/27797433-a471666e-5fdb-11e7-8779-f8650f68c10a.png) If that works for you, I'll push it